server/src/resolvers/getVariantsResolver/adapters/remoteTestNodeAdapter.ts

```diff
@@ -113,9 +113,8 @@ const getRemoteTestNodeQuery = async (args: QueryInput): Promise<VariantQueryRes
   let remoteTestNodeQueryError: RemoteTestNodeQueryError | null = null;
 
   try {
-    console.log(process.env.TEST_NODE_URL);
     remoteTestNodeQueryResponse = await axios.get<StagerVariantQueryPayload[]>(
```

Comment:
`ensembl_id` is still a field in `StagerVariantQueryPayload`, maybe consider deleting it.

Reply:
Yes, we can delete it. I've switched Stager search to purely gene-based.